xmake/core/_xmake_main.lua

```diff
@@ -31,12 +31,13 @@ xmake._NULDEV           = _NULDEV
 xmake._VERSION          = _VERSION
 xmake._VERSION_SHORT    = _VERSION_SHORT
 xmake._PROGRAM_DIR      = _PROGRAM_DIR
-xmake._PROGRAM_FILE     = _PROGRAM_FILE
+xmake._PROGRAM_FILE     = _EXECUTABLE_PATH
 xmake._PROJECT_DIR      = _PROJECT_DIR
 xmake._CORE_DIR         = _PROGRAM_DIR .. "/core"
 xmake._TOOLS_DIR        = _PROGRAM_DIR .. "/tools"
 xmake._TEMPLATES_DIR    = _PROGRAM_DIR .. "/templates"
 xmake._PROJECT_FILE     = "xmake.lua"
+xmake._EXECUTABLE_PATH  = _EXECUTABLE_PATH
```

Comment:
I think it's enough to use `xmake._PROGRAM_FILE     = _PROGRAM_FILE`, There's no need to add a redundant variable